packages/core/src/authenticatedFetch/fetchFactory.spec.ts

```diff
@@ -314,7 +330,9 @@ describe("buildAuthenticatedFetch", () => {
 
   // jest.useFakeTimers();
   function sleep(ms: number) {
-    return new Promise((resolve) => setTimeout(resolve, ms));
+    return new Promise((resolve) => {
+      setTimeout(resolve, ms);
+    });
```

Comment:
When we use fake timers, we'll need to change these calls to `jest.advanceTimersByTime(ms);`